src/App.js

```diff
@@ -1,13 +1,12 @@
 import React, { Component } from 'react';
-import Users from './components/Users/users';
+import Post from './components/Post/post';
 
 class App extends Component {
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <Users/>
-        </header>
+        <Post/>
```

Comment:
I see this element has a form.
Why not calling it `PostForm` instead of just `Post`? This to avoid making it confusing later on.

Reply:
ok